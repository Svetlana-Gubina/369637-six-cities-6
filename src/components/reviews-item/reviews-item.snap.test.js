import React from 'react';
import {render} from '@testing-library/react';
import ReviewsItem from './reviews-item';

test(`Should ReviewsItem render correctly`, () => {
  const review = {
    userAvatar: `img/1.png`,
    userName: `Oliver`,
    userRate: 3,
    reviewText: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    reviewTime: `March 2021`,
  };

  const {container} = render(<ReviewsItem userAvatar={review.userAvatar} userName={review.userName} userRate={review.userRate} reviewText={review.reviewText} reviewTime={review.reviewTime} />);
  expect(container).toMatchSnapshot();
});
