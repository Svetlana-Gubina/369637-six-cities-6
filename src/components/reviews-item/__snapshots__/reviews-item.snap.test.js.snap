// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should ReviewsItem render correctly 1`] = `
<div>
  <li
    class="reviews__item"
  >
    <div
      class="reviews__user user"
    >
      <div
        class="reviews__avatar-wrapper user__avatar-wrapper"
      >
        <img
          alt="Reviews avatar"
          class="reviews__avatar user__avatar"
          height="54"
          src="img/1.png"
          width="54"
        />
      </div>
      <span
        class="reviews__user-name"
      >
        Oliver
      </span>
    </div>
    <div
      class="reviews__info"
    >
      <div
        class="reviews__rating rating"
      >
        <div
          class="reviews__stars rating__stars"
        >
          <span
            style="width: 60%;"
          />
          <span
            class="visually-hidden"
          >
            Rating
          </span>
        </div>
      </div>
      <p
        class="reviews__text"
      >
        A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.
      </p>
      <time
        class="reviews__time"
        datetime="March 2021"
      >
        March 2021
      </time>
    </div>
  </li>
</div>
`;
